packages/core/test/CardanoNode/mocks.ts

```diff
@@ -48,11 +48,18 @@ export const mockStakeDistribution: StakeDistribution = new Map([
   ]
 ]);
 
+const projectedTip: Cardano.Tip = {
+  blockNo: Cardano.BlockNo(863),
+  hash: Cardano.BlockId('70de6ec951af0a47b000d0f2909270896d6c3d3de3951c96b0979164941bb15f'),
+  slot: Cardano.Slot(8450)
+};
+
 export const healthCheckResponseMock = (opts?: {
   blockNo?: number;
   slot?: number;
   hash?: string;
   networkSync?: Cardano.Percent;
+  withTip?: boolean;
```

Comment:
Do we have scenarios where the tip is not needed in the response?

Reply:
Yes, I had to add it in order to distinguish tests which was expecting a result with tip from those whose don't.

Reply:
:thinking: Which tests are those? I must be missing when the scenario of not having projected tip information is valid.

Reply:
@rhyslbw , my guess is: the tests which use some test `Provider` implementation rather than real `Provider`s...
but I need to check this in order to provide a real answer.

Reply:
Ah, yes!